[Term Entry] Lua Math .fmod() (#3459)
Codecademy/docs: content/lua/concepts/mathematical-library/terms/fmod/fmod.md

---

> Title: '.fmod()'

Reviewer: ```suggestion
Title: 'fmod()'
```

> Description: 'Returns the remainder of the quotient of two parameters.'

Reviewer: ```suggestion
Description: 'Returns the remainder of the quotient of two arguments.'
```

Subjects:
  - 'Computer Science'
Tags:
  - 'Arithmetic'
  - 'Functions'
  - 'Math'
  - 'Numbers'
CatalogContent:
  - 'learn-lua'
  - 'paths/computer-science'
---

> The method **`.fmod()`** takes two numbers as parameters and returns the remainder of the first number divided by the second number.

Reviewer: ```suggestion
The method **`fmod()`** takes two numbers and returns the remainder of the first number divided by the second number.
```

## Syntax

> The syntax of `.fmod()` is as follows:

Reviewer: ```suggestion
The syntax of `fmod()` is as follows:
```

```pseudo
math.fmod(num1, num2)
```

> Where `num1` names the number to be divided by `num2`.

Reviewer: ```suggestion
Where `num1` is the value to be divided by `num2`.
```

## Examples

> The following examples illustrate the behavior of `.fmod()`:

Reviewer: ```suggestion
The following examples illustrate the behavior of `fmod()`:
```
